install-manifests/enterprise/clickhouse/docker-compose.yaml

```diff
@@ -51,9 +51,9 @@ services:
     image: hasura/clickhouse-data-connector:v2.32.0
     restart: always
     ports:
-      - 8080:8081
+      - 8081:8081
     healthcheck:
-      test: ["CMD", "curl", "-f", "http://localhost:8080/health"]
+      test: ["CMD", "curl", "-f", "http://localhost:8081/health"]
```

Comment:
I think the logs in the data-connector-aggent print that the service is running at port `8080` instead of port `8081`. It should be `ports: 8081:8080`.